src/cpp/src/tokenizer.cpp

```diff
@@ -370,21 +364,6 @@ class Tokenizer::TokenizerImpl {
         }
     }
 
-    // tokenize str representation to get special tokens integer values
-    void infer_special_tokens_if_necessary() {
```

Comment:
is not it required for https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/test_tokenizer.py#L291 ?

I suppose to enable that test we still need `infer_special_tokens_if_necessary`

Reply:
Reverted